ExpressMapper.Tests NET40/CollectionTests.cs

```diff
@@ -31,6 +31,64 @@ public void AutoMemberMap()
         }
 
         [Test]
+        public void AutoMemberMap_ToNonGenericListInherited() {
+          Mapper.Register<TestModel, TestViewModel>();
+          Mapper.Register<Size, SizeViewModel>();
+          Mapper.Register<Country, CountryViewModel>();
+          Mapper.Compile();
+
+          var testData = Functional.CollectionAutoMemberMap();
+
+          var result = Mapper.Map<List<TestModel>, NonGenericCollectionInhertedFromList>( testData.Key );

```

Comment:
I think destination type should be inherited from non-generic interface! Please inherit from IList / IEnumerable or ICollection.